tools/new_minor_OCP_version.py

```diff
@@ -196,6 +227,39 @@ def change_version_in_files(old_version ,new_version):
                 file=file
             ),shell=True)
 
+def add_single_node_fake_4_8_release_image(openshift_versions_json):
+    versions = json.loads(openshift_versions_json)
```

Comment:
Consider moving this to another file ( e.g. non_production_release.images.json) and get the content from there

Reply:
Solved in 1af5150